### PR TITLE
Remove old CLI + cleanup

### PR DESCRIPTION
**Pull request by iameskild**
Fixes | Closes | Resolves #

> Please remove anything marked as optional that you don't need to fill in.
> Choose one of the keywords preceding to refer to the issue this PR solves, followed by the issue number (e.g Fixes # 666).
> If there is no issue, remove the line. Remove this note after reading.

## Changes introduced in this PR:

- Removes the old CLI from the codebase
- Update reference to logo in theme

## Types of changes

What types of changes does your PR introduce?

_Put an `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

### Requires testing

- [ ] Yes
- [ ] No

### In case you checked yes, did you write tests?

- [ ] Yes
- [ ] No

## Documentation

Does your contribution include breaking changes or deprecations?
If so have you updated the documentation?

- [ ] Yes, docstrings
- [ ] Yes, main documentation
- [ ] Yes, deprecation notices

## Further comments (optional)

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered and more.

### CODE REVIEW

File: nebari/initialize.py [hunk 1]
```diff
@@ -49,14 +49,8 @@ def base_configuration():
...
-                "logo": "/hub/custom/images/jupyter_nebari_logo.svg",
-                "primary_color": "#4f4173",
-                "secondary_color": "#957da6",
-                "accent_color": "#32C574",
-                "text_color": "#111111",
-                "h1_color": "#652e8e",
-                "h2_color": "#652e8e",
-                "version": f"v{__version__}",
+                "logo": "https://raw.githubusercontent.com/nebari-dev/nebari-design/main/logo-mark/horizontal/Nebari-Logo-Horizontal-Lockup-White-text.svg",
+                "version": f"{__version__}",
```
Comment (viniciusdc) [inline nebari/initialize.py:53]: @iameskild, you may drop the version field and switch it to `display_version.` The value is passed under `input_vars` automatically when the above is present, but there is no version assigned.

https://github.com/nebari-dev/nebari/blob/c08ee7164693ddd05f7956a932f7628bfee98ebc/qhub/stages/input_vars.py#L264

**Review by viniciusdc — APPROVED** [review]
These changes look fine to me, and I don't think they could break any major process. I will approve and test locally, but this is ready to go

**Comment by iameskild** [comment]
> These changes look fine to me, and I don't think they could break any major process. Are we testing the keycloak add user command in the tests bellow? if yes, then I would say that this is ready to go 😄

We are indeed testing adding and listing keycloak users:
https://github.com/nebari-dev/nebari/blob/c08ee7164693ddd05f7956a932f7628bfee98ebc/.github/workflows/kubernetes_test.yaml#L127

Thanks @viniciusdc 828

**Comment by viniciusdc** [comment]
@iameskild could you fix the conflict bellow? after that, this is ready for merge and we will be in place for another RC

**Pull request merged**